lib/faker/date.ex

```diff
@@ -47,15 +45,26 @@ if Version.match?(System.version(), ">= 1.3.0") do
     """
     @spec forward(integer) :: Date.t
     def forward(days) do
-      unix_now =
-        DateTime.utc_now()
-        |> DateTime.to_unix()
+      Faker.DateTime.forward(days)
+      |> DateTime.to_date
+    end
+
+    @doc """
+    Returns a random date between two dates, start date and end date not
+    included
+    """
+    @spec between(Date.t, Date.t) :: Date.t
+    def between(from, to) do
+      Faker.DateTime.between(to_datetime(from), to_datetime(to))
```

Comment:
I'm not sure I follow. I would still have to convert `Date` to `DateTime`. It would just be pushing `to_datetime` to `Faker.DateTime`. Unless we maintain two implementations of `between/2` for `DateTime.t` and `Date.t`. If that were the case I'd think it would make sense to keep the `Date.t` implementation in `Faker.Date`.

I feel like I'm missing something. Let me know.